test/verify/check-source

```diff
@@ -52,17 +52,6 @@ class TestSource(composerlib.ComposerCase):
         # groups action done
         b.click("button:contains('Add source')")
 
-        # HACK: workaround issue https://github.com/osbuild/cockpit-composer/issues/989
```

Comment:
So this commit completely fixes #989 or just one of those problematic dialogs? If all of them, then the commit message could do with `Fixes` line.

Reply:
Added fixes